misk/src/main/kotlin/misk/web/extractors/RequestBodyParameterExtractor.kt

```diff
@@ -25,7 +25,7 @@ internal class RequestBodyParameterExtractor(
   ): Any? {
     val mediaType = request.headers["Content-Type"]?.let { MediaType.parse(it) }
     val unmarshaller = mediaType?.let { type ->
-      unmarshallerFactories.map { it.create(type, parameter.type) }.firstOrNull()
+      unmarshallerFactories.map { it.create(type, parameter.type) }.filterNotNull().firstOrNull()
```

Comment:
There was a subtle bug here. `firstOrNull` returns null if the list is empty or the first element. If the list is [null, SupportedUnmarshaller.Factory], then this would return null which isn't what you want. Instead, filter the list of nulls first and then return `firstOrNull`